app/governance/proposal/page.tsx

```diff
@@ -25,11 +25,27 @@ import {
   PROPOSAL_TURNOUT_VALUE,
 } from "@/config/consts/config";
 
+import Spacer from "@/components/layout/spacer";
+import useStaking from "@/hooks/staking/useStaking";
+import { VoteBarGraph } from "../components/votingChart/voteGraph";
+
 const VOTE_OPTION_COLORS = {
-  [VoteOption.YES]: "rgb(6, 252, 153)",
```

Comment:
For the Voting option box color, added new colors from globals and border colors too.